vars/runATH.groovy

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/env groovy
+
+/**
+ * Simple wrapper for running the ATH
+ */
+
+def call(Map params = [:]) {
+
+    def String athUrl = params.get('athUrl', 'https://github.com/jenkinsci/acceptance-test-harness.git')
+    def String athRevision = params.get('athRevision', 'master')
+    def String metadataFile = params.get('metadataFile', 'essentials.yml')
+    def String jenkins = params.get('jenkins', 'latest')
+
+    def isLocalATH = athUrl.startsWith("file://")
+    def isVersionNumber = (jenkins =~ /^(\d+\.)?(\d+\.)?(\*|\d+)$/).matches()
+    // Workaround for https://issues.jenkins-ci.org/browse/JENKINS-27092
+    def shouldStop = false
+    // End of workaround
+
+    def mirror = "http://mirrors.jenkins.io/"
+    def defaultCategory = "org.jenkinsci.test.acceptance.junit.SmokeTest"
```

Comment:
There should be no default for this, I think, because that would block you from running tests without this annotation IIUC.

Reply:
That default is applied, unless a mistake on my part, only when you there are neither tests nor categories specified

```
stage("Running ATH") {
        def testsToRun = metadata.tests?.join(",")
        def categoriesToRun = metadata.categories?.join(",")
        def browsers = metadata.browsers ?: ["firefox"]
        def failFast = metadata.failFast ?: false
        def rerunCount = metadata.rerunFailingTestsCount ?: 0
        def localSnapshots = metadata.useLocalSnapshots ?: true

        if (testsToRun == null && categoriesToRun == null) {
            categoriesToRun = defaultCategory
        }
```

So you can specify any combination of individual tests and categories you want, and only in the case that you specify nothing the default is executed